atlite/resource.py

```diff
@@ -28,22 +28,38 @@
 from operator import itemgetter
 import numpy as np
 from scipy.signal import fftconvolve
-from pkg_resources import resource_stream
+from pkg_resources import resource_filename
 
 import logging
 logger = logging.getLogger(name=__name__)
 
+from . import config
+from .utils import construct_filepath
+
 def get_windturbineconfig(turbine):
     """Load the 'turbine'.yaml file from local disk and provide a turbine dict."""
 
-    res_name = "resources/windturbine/" + turbine + ".yaml"
-    turbineconf = yaml.safe_load(resource_stream(__name__, res_name))
+    res_name = os.path.join(config.windturbine_dir, turbine+".yaml")
+
+    res_name = construct_filepath(res_name)
```

Comment:
If the user does not set the `windturbine_dir` and `solarpanel_dir` correctly, then yes.

This is a side effect of the choice of how `config_path` is implicitly used.
We could
a.) Read the `default.config.yaml` if the new user's config file does not provide the variables (fall-back for `windturbine_dir` and `solarpanel_dir`.

or:

b.) Do a.) as a general fallback for all variables which are not defined in the user's `config.yaml`.

With that solution, the user would only not be able to access the turbine configs, if they explicitly overwrite the default.

Reply:
I like b.), with two small additions:
- We add `config.example.yaml` where all config handles are commented out and instruct users to copy this file instead `default.config.yaml
- We rename `default.config.yaml` to `config.default.yaml`, so that `config.{example,default}.yaml` appear next to each other

Reply:
Sounds good.
I'll also add a special prefix for the lines ('<ATLITE>') to refer to the package home,
so one does not have to rely on (guessing) absolute paths.